frontend/pages/index.js

```diff
@@ -180,6 +182,62 @@ const Homepage = () => {
       setBackgroundLoaded3840(true);
     };
   }, []);
+
+  const MilestoneBox = ({ headingText, description }) => {
+    return (
+      <Box
+        w={["150px", "180px", "300px", "300px", "400px", "500px"]}
+        // bg="gray.100"
+        py={5}
+      >
+        <Text
+          fontSize={"xl"}
+          ml={5}
+          color="orange.500"
+          fontWeight="bold"
+          textAlign="center"
+        >
+          {headingText}
+        </Text>
+        <Text textAlign="center">{description}</Text>
+      </Box>
+    );
+  };
+
+  const JoinDiscordButton = () => {
+    return (
+      <RouteButton
+        minW={["200px", "250px", "250px", "300px", "350px", "400px"]}
+        alignItems="center"
+        justifyContent="center"
+        border="solid transparent"
+        fontWeight="bold"
+        rounded={["lg", "xl", "2xl"]}
+        shadow="md"
+        variant="solid"
+        colorScheme="orange"
+        textColor="blue.1200"
+        fontSize={["lg", "xl", "2xl", "3xl", "4xl", "4xl"]}
+        py={[4, 6, 6, 8, 8]}
+        px={[4, 4, 4, 8, 8]}
+        onClick={() => {
```

Comment:
OnClick should be realized in JoinDiscordButton implementation

Reply:
Expected code would look like 
page
```jsx
<Button variant="DiscordButton" onClick={() => { mixpanel.track()}}>Join us in discord</Button>
```

That way also you can add mixpanel track property to identify from which section button was pressed